modules/rostests/apitests/user32/keybd_event.c

```diff
@@ -0,0 +1,35 @@
+#include "precomp.h"
+
+void testScancodeExtendedKey(BYTE wVk, BYTE scanCode)
+{
+	trace("wVK: %x\tScancode: %x\n", wVk, scanCode);
+
+	keybd_event(0, scanCode, KEYEVENTF_SCANCODE | KEYEVENTF_EXTENDEDKEY, 0);
+    SHORT winKeyState = GetAsyncKeyState(wVk);
```

Comment:
Your indentation suddenly switches from 2 to 4 spaces here. Please keep it consistent (ideally at 4 everywhere).